css/at-rules/media.json

```
@@ -1375,9 +1375,16 @@
                 "web-features:prefers-color-scheme"
               ],
               "support": {
-                "chrome": {
-                  "version_added": false
-                },
+                "chrome": [
+                  {
+                    "version_added": "129"
+                  },
+                  {
+                    "version_added": "111",
+                    "partial_implementation": true,
+                    "notes": "Only supports SVG images, not iframes"
+                  }
```

Nit: End with period

```suggestion
                    "notes": "Only supports SVG images, not iframes."
```

---

Sorry, VSCode put the suggestion on the wrong line. 🤷